src/WeightUnit.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration;
+
+use MyCLabs\Enum\Enum;
```

Comment:
Is there a reason why this project still uses php 8.0?

Reply:
Is, the integrations are not upgraded to PHP 8.2 yet. When we upgrade them we can come back here and use proper enum, also readonly props.